src/UdpDataProtocol.cpp

```diff
@@ -388,7 +337,8 @@ functions. DWORD n_bytes; WSABUF buffer; int error; buffer.len = n; buffer.buf =
         n_bytes = ::sendto(mSocket, buf, n, 0, (struct sockaddr*)&mPeerAddr6,
                            sizeof(mPeerAddr6));
     } else {
-        n_bytes = ::send(mSocket, buf, n, 0);
+        n_bytes =
+            ::sendto(mSocket, buf, n, 0, (struct sockaddr*)&mPeerAddr, sizeof(mPeerAddr));
```

Comment:
Both times we cast to struct sockaddr*. Could it be that we don't need to distinguish between IPv4 and IPv6 at this point?

Reply:
Nevermind.